docs/src/install.rst

```diff
@@ -1,14 +1,30 @@
 Build and Install
 =================
 
+Install from PyPI
+-----------------
+
+MLX Data can be installed from PyPI. For most Linuxes and Apple silicon Macs
+one can simply,
+
+.. code-block::
+
+    pip install mlx-data
+
+This should install the package together with the necessary dependencies to
+read audio, images, video and remote content from S3.
+
+Building from source
+--------------------
+
 MLX Data consists of a C++ library with shallow python bindings and some Python
 helpers. You can choose to build and use only the C++ library or simply pip
 install the Python library which transparently builds the required C++ backend.
 
 We plan to provide prebuilt binaries for easy install in the future.
 
 Dependencies
-------------
+^^^^^^^^^^^^
```

Comment:
Won't some of the dependencies build automatically with the cmake?

Reply:
In fact, aws is the only one that doesn't? If I understand correctly, we should simplify the paragraphs below to just have a line on using aws.

Reply:
None except if one is using the super build. I still think it is valid for people to dynamically link if they are building on their own. It is much faster and allows building with many things missing.

I did fix the docs though. I had forgotten to update the old part of the installation. Let me know if it needs a few more clarifications.